packages/api/src/store/table.ts

```diff
@@ -15,7 +15,7 @@ import {
   FieldSpec,
 } from "./types";
 
-const DEFAULT_SORT = "id ASC";
```

Comment:
FTR: This was not really worth it. It (luckily) broke tests and when I went to do some further testing I noticed it's not really making queries faster (sometimes they're worse, cause not all objects have an indexed `data->>'id'`). Let's stick to just the `id` column.